docs/documentation/pages/DECKHOUSE-FAQ.md

```diff
@@ -166,6 +167,7 @@ To switch the Deckhouse cluster to using a third-party registry, follow these st
 
 * Update the `image` field in the `d8-system/deckhouse` deployment to contain the address of the Deckhouse image in the third-party-registry;
 * Edit the `d8-system/deckhouse-registry` secret (note that all parameters are Base64-encoded):
+  * To correctly write values to secrets without use: `echo -n 'value' | base64`;
```

Comment:
Changes are obsolete. To switch the Deckhouse cluster to using a third-party registry, [a script](https://deckhouse.io/documentation/v1/deckhouse-faq.html#how-do-i-switch-a-running-deckhouse-cluster-to-use-a-third-party-registry) is used.